uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/BAO/Assignment.php

```diff
@@ -24,36 +24,26 @@ public static function create($params) {
 
 
   /**
-   * Get all cases with no end dates
+   * retrieve cases along with related contact details
    *
-   * @param array $params
+   * @param array $params accept 'sort_name' and 'limit'
```

Comment:
maybe in the future a new parameters will be added and used for othr sorts of  filtering , so I would keep it like than rather than keep adding new parameters to the method itself.